sim/firesim-lib/src/main/scala/bridges/TracerVBridge.scala

```diff
@@ -247,8 +239,8 @@ class TracerVBridgeModule(key: TraceBundleWidths)(implicit p: Parameters)
     val maybeFire = !anyValidRemainMux || (counter === (armCount - 1).U)
     val maybeEnq  = anyValidRemainMux
 
-    val do_enq_helper  = DecoupledHelper(hPort.toHost.hValid, streamEnq.ready, maybeEnq, traceEnable)
-    val do_fire_helper = DecoupledHelper(hPort.toHost.hValid, streamEnq.ready, maybeFire)
+    val do_enq_helper  = DecoupledHelper(hPort.toHost.hValid, hPort.fromHost.hReady, streamEnq.ready, maybeEnq, traceEnable)
+    val do_fire_helper = DecoupledHelper(hPort.toHost.hValid, hPort.fromHost.hReady, streamEnq.ready, maybeFire)
```

Comment:
Shouldn't the `initDone` signal also be added here s.t. spurious tokens don't get sent into the target or to the DMA queue?

Reply:
For example, the prior helper had it: https://github.com/firesim/firesim/pull/1547/files#diff-302883d7fe8affa977e8dc737cf61f23183397f2f1548d5fab6438dd76e13501L204